ci/ray_ci/test_tester.py

```diff
@@ -144,11 +159,24 @@ def test_get_all_test_query() -> None:
 def test_get_flaky_test_targets() -> None:
     _TEST_YAML = "flaky_tests: [windows://t1, //t2]"
 
-    with TemporaryDirectory() as tmp:
+    with TemporaryDirectory() as tmp, mock.patch(
+        "ray_release.test.Test.gen_from_s3",
+        return_value=[
+            _stub_test({"name": "windows://t1_s3", "team": "core"}),
+            _stub_test({"name": "windows://t2_s3", "team": "ci"}),
+            _stub_test({"name": "linux://t3_s3", "team": "core"}),
+        ],
+    ):
```

Comment:
nit: can we rename `_TEST_YAML` into `_FAKE_CORE_TEST_YAML`, or something that indicates it is the yaml file for core team?